routes/scripts.go

```diff
@@ -0,0 +1,48 @@
+package routes
+
+import (
+	"github.com/labstack/echo"
+	"github.com/quicky-dev/hub-api/controllers"
+)
+
+/* ----------------------------- Linux Packages ----------------------------- */
+
+func GetUbuntuGeneric(app *echo.Echo) {
```

Comment:
Collapse these functions into one single function that registers the routes like so:

```go
func RegisterScriptRoutes (app *echo.Echo) {
    app.GET(...)
    app.GET(...)
    ...
}